source/release-notes/1.5.txt

```diff
@@ -0,0 +1,58 @@
+.. _c2c-release-notes-1.5:
+
+===================================
+Release Notes for mongosync 1.5
+===================================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.5.0-c2c-release-notes:
+
+1.5.0 Release
+-------------
+
+**July 25, 2023**
+
+Oplog Rollover Resiliency
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
(versioning)
You might want to include 1.5.0 here, since there are likely to be more 1.5.x releases going forward. 
Global

Reply:
Fixed